index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Art Exhibition: Home page</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-light bgp-light">
+      <div class="container-fluid">
+        <a class="navbar-brand p-0 m-0" href="#"><img class="p-0 m-0 logo-container"
+            src="./img/log.jpg" alt="icon"></a>
+        <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page"
+                href="./about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">News</a>
+            </li>
+            <li class="nav-item mx-md-5 nav-content">
+              <a class="nav-link">CC Campain</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
```

Comment:
- [ ] According to the original design, The navbar in `desktop version` is divided into a top dark bar and a bottom white bar, I think you forgot to add The top short bar section with social media links, logout, and language links aligned to the right, so Kindly add this part 👌

- [ ] Remember to implement this part on both (the Home and About pages) 💯

| Your Design  | Orignal Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181183490-a5c5e658-a5c6-4b66-abc0-4c698647847e.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181183561-1fbfdfe5-56ab-48e0-a51c-7b40c4c93155.PNG" width="500" /> |

![image](https://user-images.githubusercontent.com/45231276/181183829-ad3646d1-29a5-48c6-97ff-a1ae6365877b.png)